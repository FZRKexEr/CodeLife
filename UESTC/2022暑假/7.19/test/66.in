10
jswgfcoaatxqgzyjumvhfzpejpwcbjonlazwsslyzmbxnqshpfxrsqkkwgscanaoiszgirlglcnzrsxepzwwgevcflnahtmcffqk
9
stsgkokxqwgnpfxzhjuxljgrxjngweshitmujhpgykevhrrzsvynctywuyxwduiecfcvivojvjkrmgxogkkylkgozncdhagmvzsb
1
wxiwgbgmwewwrjxxvzkvcdudrmvimtgbpvhwxffilqiniitherbkamydohsqubgsgsyehnuuqcuiafvhiuwzhpyksmjuvqgiyoft
2
ftgcndrbpfdguklzqplcbthklpwpnnhppcabfzdcniuyczmgzabjshepgcstqkmmoloomvaviisxkhtuxklnqxhinayahjrtkcsn
1
tmwrdzzuhytrnlrirtyreckurerumryqzlaioeqkhdkrgeumdpzienpkyrxwpdptgzmrogscxsyggqxktiqcirnqfcseeplgzwce
6
wlgavcoeepxvwpbhtgdbqugsytlhkvyhurmmcokmjdmrwajunmifvbpqkfygqmfiabbbhevptyjrziyzkobfudztflhedbsfvkri
9
ubyzvthjtfzxjftfgxjrppxdibdgyjatpuudnzbbwtfvkiubjrppyewazlyepeaifkqbjiciknwmnnaponlqdxlzvicgsostxuif
5
qiujaiagbxeyplnuuavsdikarpjhbbbtkkeioftnzqykyaghjcdkfwloradaqfjaxhybqpniwrcbmjlxvjwgksqtinkvkpglauih
8
srxfdlstthvuelwpqyxnkdkkxdsilqotmdjvlvdpixotszhyxlvlricjgfwnxznlxpzjeyxirjagxbhlsoypfhvthrzghuakwimz
3
beebmznkubfbrlcpgebennqjpyjypttivijzuamtiynqvqemyhryutkyjgmqawdizpkfsbeujepisifaotylueggepzrpbieqzwr
4
