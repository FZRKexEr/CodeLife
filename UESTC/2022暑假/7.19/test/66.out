2048
882499718
65980984
882499718
65536
2048
1048576
4096
603979768
33554432
